### ISSUE TITLE
New headless version is not compatible with puppeteer's bin (error while loading shared libraries: libXss.so)

### ISSUE THREAD

**Issue by MLefebvreICO** [sylvaindumont/docker-node-karma-protractor-chrome#28]
Hi there,

So with the new version, we cannot use `puppeteer` anymore, it will fail to start chrome saying `libXss.so` is missing.
![image](https://user-images.githubusercontent.com/17457252/104234646-b9a22f80-5421-11eb-8150-816b2241b49a.png)

What up with that ?

We are using `karma` with `puppeteer`'s bin (`process.env.CHROME_BIN = require('puppeteer').executablePath();`) with the image `weboaks/node-karma-protractor-chrome:headless` in our `.gitlab-ci.yml`.

We use `puppeteer`'s bin to make sure we are testing against the same Chrome version as our PCs. 
It was working before the newest publish, but now it won't start the browser at all .. 😐 

Thanks for your help !

**Comment by sylvaindumont** [comment]
Hi,
I had the same problem with an old puppeteer version embedded in Penthouse, but  I wasn't sure other users of this image were using puppeteer too, so I just upgraded my puppeteer version instead of fixing the image for the old chromium versions embedded in puppeteer...
I just pushed a fix to the five Debian variants of the image fixing its use with old chromium versions. Let me know if the latest `headless` fixes the issue for you.
Thank you for the detailed report!
